### DISCUSSION TITLE
How to get width and height of svg content

### DISCUSSION THREAD

**Discussion by gnvcor** [fabricjs/fabric.js#8467] [Q&A] [answered]
Example:

Here, using `canvas.toSVG();` i get svg with area where it was drawn:
<img width="151" alt="image" src="https://user-images.githubusercontent.com/8590608/203906257-98782661-b32d-4fe4-85ad-3301c56c859e.png">

But, I need to get svg only with width and height:
<img width="107" alt="image" src="https://user-images.githubusercontent.com/8590608/203906312-a4547a03-60c0-422b-bc0b-14e4c6f95e45.png">

For example, [paper.js](https://github.com/paperjs/paper.js) has two methods for exporting svg:
1. `exportSVG({ bounds: 'view' })` -- I get svg with view, as in screenshot 1
2. `exportSVG({ bounds: 'content' })` -- I get svg without view, only content, as in screenshot 2

**Comment by ShaMan123** [comment, marked as answer]
You should do `object.toSVG()`

**Reply by gnvcor** [reply]
@ShaMan123 yes, i do, but i get svg with view
![image](https://user-images.githubusercontent.com/8590608/204011378-d446aecc-c90a-49f4-bfca-03db24339ab0.png)

I want get svg content
![image](https://user-images.githubusercontent.com/8590608/204011488-c5b21842-8270-4a56-ac30-4b3641fada1b.png)

**Reply by ShaMan123** [reply]
Pay attention.
Not canvas.toSVG 
OBJECT.toSVG

**Reply by gnvcor** [reply]
@ShaMan123 `canvas` in example as `fabric.Canvas` type.

I know how get svg in fabricjs.

I don't now, how get svg content (without view).

**Reply by ShaMan123** [reply]
So as I said, you should call `myObject.toSVG()` as in `path.toSVG()`
I can't make it any clearer or simpler.
